js_basics/task1.js

```diff
@@ -0,0 +1,12 @@
+let stringValue = "new line";
+let numberValue = 26;
+let booleanValue = true;
+let nullValue = null;
+let undefinedValue;
+
+console.log("This is string: <" + stringValue + ">.");
```

Comment:
Please print all values separately e.g. 
`console.log("This is string:", stringValue)`

Now all values you have printed have type string (due to concatenation)